docs/how_to/single_node.md

```diff
@@ -29,7 +29,7 @@ Next, create an initial namespace for your metrics:
 <!-- TODO: link to config reference docs once available -->
 
 ```json
-curl -X POST localhost:7201/namespace/add -d '{
+curl -X POST localhost:7201/api/v1/namespace/add -d '{
```

Comment:
mind checking if you need to make similar changes to Matt's kube guide too

Reply:
Done.